<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-U8JJ59ON" failures="1" tests="2" name="Default test(failed)" time="0.128" errors="0" timestamp="2025-07-02T23:12:12 IST">
  <testcase classname="demo.TestNGRetryFailureDemo" name="Test2" time="0.017"/>
  <testcase classname="demo.TestNGRetryFailureDemo" name="Test3" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailureDemo.Test3(TestNGRetryFailureDemo.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="demo.TestNGRetryFailureDemo" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Default test(failed) -->
